sirbot-pyslackers/endpoints/slack/messages.py

```diff
@@ -31,6 +31,88 @@ def create_endpoints(plugin):
     # stock tickers are 1-5 capital characters, with a dot allowed. To keep
     # this from triggering with random text we require a leading '$'
     plugin.on_message("s\$[A-Z\.]{1,5}", stock_quote, wait=False)
+    plugin.on_message("c\$[A-Z\.]{1,5}", crypto_quote, wait=False)
+
+
+async def crypto_quote(message, app):
+    match = STOCK_REGEX.search(message.get("text", ""))
+    if not match:
+        return
+
+    symbol = match.group("symbol")
+    LOG.debug("Fetching crypto quotes for symbol %s", symbol)
+
+    response = message.response()
+    try:
+        stocks = app["plugins"]["stocks"]
+        crypto = await stocks.crypto()
+        quote = None
+        for c in crypto:
+            if c["symbol"] == f"{symbol}USDT":
+                quote = c
+                break
+        if quote is None:
+            response["text"] = f"The crypto symbol `{symbol}` could not be found"
+            LOG.error("No crypto currencies found when searching for: %s", symbol)
```

Comment:
I would not do a `ERROR` here since it will probably happens time to time when people make typo or search for a non existing currencies.

AFAIK `ERROR` are sent to sentry

Reply:
I'll go with `debug` then. Thanks!